spec/models/concerns/github_teamable_spec.rb

```diff
@@ -0,0 +1,98 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+class MockGitHubTeam
+  attr_reader :id
+
+  def initialize(id)
+    @id = id
+  end
+end
+
+shared_examples_for "github_teamable" do
```

Comment:
Added these tests because this `ActiveSupport::Concern` didn't have any test coverage.

Reply:
🙌